backend/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "nerds-against-society.backend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "ts-node-dev --respawn --pretty --transpile-only src/index.ts",
+    "build": "tsc --project ./",
+    "test": "jest",
+    "prepare": "cd .. && husky install backend/.husky"
```

Comment:
It'd be nice to have a prettier/linter script I can just run in here

Reply:
No worries! I've implemented this in 80d7076